app/index.js

```diff
@@ -107,6 +111,19 @@ var NodeModuleGenerator = yeoman.Base.extend({
                 type: 'input',
                 name: 'username',
                 message: 'What is your GitHub username?',
+                store: true,
+                validate: function(input) {
+                    var res =   !input.startsWith('_')   &&
```

Comment:
I can accept that a single Regex isn't particularly more readable - it only really serves to limit the rows of code here, and that's not a big victory.